javascript/javascript2/week1/shortestWord.js

```diff
@@ -0,0 +1,12 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function shortestWord(words) {
+  let results = words[0];
```

Comment:
Try to add meaningfull names for your variables. See bellow
```suggestion
  let shortestWord= words[0];
```